network/network.go

```diff
@@ -124,6 +125,9 @@ func New(log zerolog.Logger, codec Codec, options ...func(*Config)) Network {
 	dialer := &simpleDialWrapper{}
 	net.dialer = dialer
 
+	eventMgr := &simpleEventManager{subscriber: subscriber}
+	net.eventMgr = eventMgr
```

Comment:
Simpler naming would be to simply call it `events`, it reads nicely that way as well:
```
events.Disconnected(address)
```